Fedoraware/TeamFortress2/TeamFortress2/Utils/Utils.h

```diff
@@ -28,9 +28,6 @@
 #undef min
 #undef max
 
-#define CONTINUE_IF(cond) if (cond) { continue; }
```

Comment:
why

Reply:
i think it wasn't a good idea anymore...

Reply:
fairs